pyproject.toml

```diff
@@ -38,7 +38,9 @@ include = [
 ]
 requires-python = ">=3.9.0"
 
-dependencies = []
+dependencies = [
+    "importlib_metadata; python_version < '3.10'",
```

Comment:
Should we add a lower bound on the version that is needed?

Reply:
- The package [`importlib.medatata`](https://docs.python.org/3.10/library/importlib.metadata.html) is in the stdlib since Python 3.10.
- The package [`importlib_metadata`](https://importlib-metadata.readthedocs.io/) is a backport for older Python version.

The latter cover all the versions that your package supports, so I don't think you need a lower bound.